content/en/docs/setup/install/helm/index.md

```diff
@@ -283,7 +175,7 @@ $ helm install install/kubernetes/helm/istio --name istio --namespace istio-syst
 
 ## Uninstall
 
-- If you installed Istio using the `helm template` command, uninstall with these commands:
+- The `helm template` command is also used in uninstalling Istio. Uninstall with these commands:
```

Comment:
We recommend avoiding passive voice ("is also used"). (https://developers.google.com/style/voice)

And it's unclear in this context what the word "also" refers to.

So I would say something like "You can use the `helm template` command to uninstall Istio."

Reply:
Resolved with https://github.com/istio/istio.io/pull/6847/commits/9381d044356a14493fa33949f50f5c90584581e0